simple_network_sim/network_of_populations.py

```diff
@@ -139,6 +208,19 @@ def distributeInfections(nodeState, newInfections):
 
 
 def doIncomingInfectionsByNode(graph, currentState, movementMultiplier):
+    """Determine the number of new infections at each node of a graph.
```

Comment:
I'd add something like *based on incoming people from neighboring nodes* just to make it extra clear that this function deals with between nodes infections.